### PR TITLE
fix(BlockHeader): refactoring

### PR DESCRIPTION
**Pull request by yuberdysheva**

### CODE REVIEW

File: src/components/HeaderComponent/__stories__/HeaderComponent.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import {Meta, Story} from '@storybook/react/types-6-0';
+import {ClassNameProps, TitleProps} from '../../../models';
+import React from 'react';
```
Comment (niktverd) [inline src/components/HeaderComponent/__stories__/HeaderComponent.stories.tsx:3]: Above local imports

File: src/text-transform/blocks.ts [hunk 2]
```diff
@@ -229,15 +216,33 @@ const config: BlocksConfig = {
...
-    [BlockType.PromoFeaturesBlock]: {
-        fields: ['items'],
-        transformer: yfmTransformer,
-        parser: parsePromoFeatures,
-    },
-    [BlockType.SliderBlock]: {
-        transformer: typografTransformer,
-        parser: parseSlider,
-    },
+    [BlockType.PromoFeaturesBlock]: [
```
Comment (gorgeousvlad) [inline src/text-transform/blocks.ts:219]:
```
const blockHeaderTransfomer =  [
       {
            fields: ['title'],
            transformer: typografTransformer,
            parser: parseTitle,
        },
        {
            fields: ['description'],
            transformer: yfmTransformer,
        }
];
```

**Review by gorgeousvlad — CHANGES_REQUESTED** [review]

File: src/blocks/CardLayout/CardLayout.tsx [hunk 3]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import {block} from '../../utils';
 import {CardLayoutBlockProps as CardLayoutBlockParams} from '../../models';
 import {Row, Col} from '../../grid';
-import {BlockHeader, AnimateBlock} from '../../components';
+import {HeaderComponent, AnimateBlock} from '../../components';
```
[the remark this answers is not in the document]
Comment (yuberdysheva) [reply src/blocks/CardLayout/CardLayout.tsx:6]: For me it is confusing that we have `HeaderBlock` and `BlockHeader`. I thought that `HeaderBlock` and `HeaderComponent` more understandable. How `BlockHeader` tells about purpose?

[the remark this answers is not in the document]
Comment (gorgeousvlad) [reply src/blocks/CardLayout/CardLayout.tsx:6]: `BlockHeader` tells that it is using for heading in blocks, `HeaderComponent` not, but raises questions - why do we have both component and block headers? 
And `HeaderComponent` has extra info in its name - we import it form `components` dir and it is clear that it is a component. And in this case should we add `Component` prefix to any component?

For to make things clear with this renaming we could exclude 'header' and use `BlockTitle` or something like that.

Anyway, we can leave this renaming for major update - it doesn't worth major with itself.

[the remark this answers is not in the document]
Comment (yuberdysheva) [reply src/blocks/CardLayout/CardLayout.tsx:6]: I like BlockTitle, let's leave this for major update. Thanks

**Review by gorgeousvlad — APPROVED** [review]

**Review by niktverd — APPROVED** [review]

**Pull request merged**